org.lflang/src/org/lflang/FileConfig.java

```diff
@@ -175,7 +175,7 @@ public FileConfig(Resource resource, IFileSystemAccess2 fsa, IGeneratorContext c
         this.srcGenPath = getSrcGenPath(this.srcGenBasePath, this.srcPkgPath,
                 this.srcPath, name);
         this.srcGenPkgPath = this.srcGenPath;
-        this.outPath = srcGenBasePath.resolve("..");
+        this.outPath = srcGenBasePath.getParent();
```

Comment:
In #468 you also changed line 204, maybe it should be updated too?

Reply:
Good catch. Thanks!